pkg/innerring/processors/netmap/nodevalidation/subnet/calls.go

```diff
@@ -0,0 +1,41 @@
+package subnet
+
+import (
+	"fmt"
+
+	morphsubnet "github.com/nspcc-dev/neofs-node/pkg/morph/client/subnet"
+	"github.com/nspcc-dev/neofs-sdk-go/netmap"
+	subnetid "github.com/nspcc-dev/neofs-sdk-go/subnet/id"
+)
+
+// VerifyAndUpdate calls subnet contract's `NodeAllowed` method.
+// Deletes attribute that has not been approved by the contract.
```

Comment:
```suggestion
// Removes subnets which has not been approved by the contract.
```
I don't see any attributes even in the function body.

Reply:
fixed